### PR TITLE
Fixes #2431 Default avatar broken on ACP when using a full URL

### LINKED ISSUE #2431: Default avatar broken on ACP when using a full URL
The code doesn't take into account that the path to the default avatar can be a full URL, too:
```php
		$user['avatar'] = "../".$mybb->settings['useravatar'];
```

### PR DESCRIPTION
**Pull request by Stefan-ST**
Fixes #2431

### CODE REVIEW

File: admin/modules/user/users.php [hunk 1]
```diff
@@ -992,7 +992,14 @@
 	}
 	else
 	{
-		$user['avatar'] = "../".str_replace('{theme}', 'images', $mybb->settings['useravatar']);
+		if(my_substr($mybb->settings['useravatar'], 0, 7) === 'http://' || my_substr($mybb->settings['useravatar'], 0, 8) === 'https://')
```
Comment (Destroy666x) [inline admin/modules/user/users.php]: Good catch, but any reason to use `my_substr()` for comparison with a plain ASCII string?

Comment (Stefan-ST) [inline admin/modules/user/users.php]: I just copied the existing code a few lines above. ;)

File: admin/modules/user/users.php [hunk 2]
```diff
@@ -992,7 +992,14 @@
 	}
 	else
 	{
-		$user['avatar'] = "../".str_replace('{theme}', 'images', $mybb->settings['useravatar']);
```
Comment (dequeues) [inline admin/modules/user/users.php]: Why did you remove this? see #2363

File: admin/modules/user/users.php (hunk 1 above)
Comment (Destroy666x) [inline admin/modules/user/users.php]: Copying doesn't explain why it's correct. But it'll be changed to the new URL function soon so doesn't matter.

File: admin/modules/user/users.php (hunk 2 above)
Comment (Stefan-ST) [inline admin/modules/user/users.php]: Sorry, fixed.

File: admin/modules/user/users.php [hunk 3]
```diff
@@ -992,7 +992,14 @@
...
+		if(my_substr($mybb->settings['useravatar'], 0, 7) === 'http://' || my_substr($mybb->settings['useravatar'], 0, 8) === 'https://')
+		{
+			$user['avatar'] = str_replace('{theme}', 'images', $mybb->settings['useravatar'])
```
Comment (dequeues) [inline admin/modules/user/users.php]: Missing `;`

Comment (Stefan-ST) [inline admin/modules/user/users.php]: Fixed, thanks.

**Comment by dequeues** [comment]
[my_validate_url()](https://github.com/mybb/mybb/blob/bcb8bd924ea6cb1a309803cf8ce730782bad70f3/inc/functions.php#L8429) can be used now rather than my_substr() for URL

**Pull request merged**